backend/libraries/utils/src/canister/update_settings.rs

```diff
@@ -0,0 +1,48 @@
+use crate::canister;
+use candid::{CandidType, Principal};
+use ic_cdk::api;
+use serde::{Deserialize, Serialize};
+use tracing::error;
+use types::CanisterId;
+
+pub async fn update_settings(canister_id: CanisterId, controllers: Vec<Principal>) -> Result<(), canister::Error> {
```

Comment:
I think this should either take the full `CanisterSettings` or should be renamed to `set_controllers`.

Reply:
Oh yes indeed I had meant to name it set_controllers